backend/src/api/task/task.controller.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  UseGuards,
+  Req,
+} from '@nestjs/common';
+import { TaskService } from './task.service';
+import { CreateTaskDto } from './dto/create-task.dto';
+import { UpdateTaskDto } from './dto/update-task.dto';
+import { ApiOkResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { JwtGuard } from '../../security/JwtGuard';
+import { User } from '../../database/entities/user.entity';
+import { Task } from '../../database/entities/task.entity';
+import { TaskResponse } from './response/task.response';
+
+@ApiTags('Task')
+@Controller('task')
+export class TaskController {
+  constructor(private readonly taskService: TaskService) {}
+
+  @Post()
+  @UseGuards(JwtGuard)
+  @ApiOkResponse({ type: TaskResponse })
+  @ApiOperation({ summary: 'Create task' })
+  create(
+    @Body() body: CreateTaskDto,
+    @Req() request: { user: User },
+  ): Promise<Task> {
+    return this.taskService.create(body, request.user);
+  }
+
+  @Get()
```

Comment:
We should return task to a user. Not all. You have to get from the request user id
request.user.id and by this id return to a user his tasks. Maybe it would be better to make two endpoints: one for returning tasks for a single user and one for getting a all tasks of his employee for boss. You can add decorator for roles, that will be working with authGuard